includes/pointers.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Notes in a Pointer dialog box for guiding users in the dashboard interface.
+ */
+add_action( 'admin_enqueue_scripts', 'pmpro_enqueue_admin_scripts' );
+/**
+ * [pmpro_enqueue_admin_scripts] Enqueue the scripts needed to builder admin pointers.
+ * 
+ * @return void
+ */
+function pmpro_enqueue_admin_scripts() {
+	wp_enqueue_style( 'wp-pointer' );
+	wp_enqueue_script( 'wp-pointer' );
+	// hook the pointer
+	add_action( 'admin_print_footer_scripts', 'prepare_pointer_script' );
+}
+/**
+ * [prepare_pointer_script] Details about PMPro 2.0 that are added to the Admin Pointer
+ * 
+ * @return void
+ */
+function prepare_pointer_script() {
+	$show_pointer = true;
+	$file_error   = true;
+
+	$id       = '#toplevel_page_pmpro-dashboard';
```

Comment:
we would put this stuff into an array if/when we have multiple pointers, but okay as is for now